### PR TITLE
Add candidates table

### PR DESCRIPTION
**Pull request by inesrosete**

### CODE REVIEW

File: db/migrate/20190918220809_create_candidates.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CreateCandidates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :candidates do |t|
+      t.string :name
+      t.string :district
+      t.string :photo
+      t.text :description
+      t.string :description_source
+      t.boolean :isLeadCandidate
```
Comment (punnie) [inline db/migrate/20190918220809_create_candidates.rb:9]: Ruby convention is snake case, therefore: `:is_lead_candidate` is preferred.

File: db/migrate/20190918220809_create_candidates.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :candidateType
```
Comment (punnie) [inline db/migrate/20190918220809_create_candidates.rb:10]: Ruby convention is snake case, therefore: `:candidate_type` is preferred.

**Review by punnie — CHANGES_REQUESTED** [review]
The comments I left are nice to haves.

However, you need to add the `party_id` to the candidate table or the association of the candidate with the party won't work.

You can achieve this by adding a `references` field on the `candidates` table, this way:

```ruby
def change
  add_reference :candidates, :party, index: true
end
```

**Comment by inesrosete** [comment]
@punnie Candidates table doesn't show in the admin page, do you know why?

**Review by punnie — APPROVED** [review]

**Pull request merged**